src/puget/printer.clj

```diff
@@ -35,6 +35,10 @@
   The text placed between a map key and a collection value. The keyword :line
   will cause line breaks if the whole map does not fit on a single line.
 
+  `:print-fallback`
+  If true, falls back to to using pr-str rather than puget's default
+  unknown-document representation
```

Comment:
Oh, I had assumed we'd literally pass the print function, e.g. your *options* would include something like:
```clojure
:print-fallback pr-str
```